docs/manualGrading.md

```diff
@@ -67,3 +67,13 @@ After editing the percentage score and/or feedback for each submitted answer, up
 Each question will have its score and/or feedback updated and the total assessment score will be recalculated. All updates are done with `credit` of 100%, so students get exactly the scores as uploaded.
 
 If you prefer to use points rather than a percentage score, rename the `score_perc` column in the CSV file to `points`.
+
+You also have the option to set partial scores. These can be based on individual elements of the question (typically based on the `answers-name` attribute of the element), or any other setting you wish to use. Partial scores must be represented using a JSON object, with keys corresponding to individual elements. Each element key should be mapped to an object, and should ideally contain values for `score` (with a value between 0 and 1) and `weight` (which defaults to 1 if not present). For example, to assign grades to a question with elements `answer1` and `answer2`, use:
+
+```json
+{"answer1": {"score": 0.7, "weight": 2, "feedback", "Almost there!"}, "answer2": {"score": 1, "weight": 1, "feedback": "Great job!"}}
+```
+
+If the `partial_scores` column contains a valid value, and there is no value in `score_perc` or `points`, the score will be computed based on the weighted average of the partial scores. For example, the score above will be computed as 80% (the weighted average between 70% with weight 2, and 100% with weight 1).
+
+*WARNING*: note that some elements such as drawings or matrix elements may rely elaborate on partial score values with specific structures and objects. When updating partial scores, make sure you follow the same structure as the original partial scores to avoid any problems. Changing these values could lead to errors on rendering the question pages for these elements.
```

Comment:
```suggestion
*WARNING*: note that some elements such as drawings or matrix elements may rely on elaborate partial score values with specific structures and objects. When updating partial scores, make sure you follow the same structure as the original partial scores to avoid any problems. Changing these values could lead to errors on rendering the question pages for these elements.
```